src/app.py

```diff
@@ -10,6 +10,14 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+garmin_s3_bucket = os.environ['GARMIN_S3_BUCKET']
+
+garmin_user = get_secret_value(os.environ['GARMIN_USERNAME'])
+garmin_pass = get_secret_value(os.environ['GARMIN_PASSWORD'])
+
+garth_client = Client()
+garth_client.login(garmin_user, garmin_pass)
+
```

Comment:
If we move these outside the handler function they should still work but they won't have to be called with every invocation of the function, just created during the warm up of a particular lambda instance.